docs/playbooks/concepts/run/index.md

```diff
@@ -0,0 +1,28 @@
+# Playbook Run
```

Comment:
merge this into the main index.md, with statues going after the main spec table

Reply:
I think it warrants its own page. (Same with approval)